### DISCUSSION TITLE
What about headers ?

### DISCUSSION THREAD

**Discussion by abdfnx** [abdfnx/resto#15] [General]
the user needs to add headers to the request

some of headers

* `Accept`
* `Accept-Encoding`
* `User-Agent`
* `Connection`
* `Cache-Control`
and so on

we're going to create a form and inside it there's a dropdown with headers

👍

**Comment by david-tomson** [comment]
it's easy, let's do it

**Comment by Timothee-Cardoso** [comment]
ok

**Comment by abdfnx** [comment]
done, it's great
